docs/source/support/must-gather.md

```diff
@@ -0,0 +1,63 @@
+# Gathering data with must-gather
+
+`must-gather` is an embedded tool in Scylla Operator that helps collecting all the necessary info when something goes wrong.
+
+The tool talks to the Kubernetes API, retrieves a predefined set of resources and saves them into a folder in your current directory.
+By default, all collected Secrets are censored to avoid sending sensitive data inside the archive.
```

Comment:
sending inside the archive sounds a bit off

Reply:
rephrased